src/docs/01_introduction_and_goals.adoc

```diff
@@ -63,6 +81,8 @@ Table with role names, person names, and their expectations with respect to the
 [options="header",cols="1,2,2"]
 |===
 |Role/Name|Contact|Expectations
-| _<Role-1>_ | _<Contact-1>_ | _<Expectation-1>_
-| _<Role-2>_ | _<Contact-2>_ | _<Expectation-2>_
+| Tim Berners-Lee | timbl@w3.org | Carry out its idea, SOLID, with the support of software developers creating applications using this technology and the Inrupt company.
```

Comment:
Tim Berners-Lee is not a "thing" so "Carry out its idea" should be "Carry out his idea"